src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+@import 'variables';
+
+body,
+ul,
+h2,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  background-color: #cdcccc;
+}
+
+.product-card {
+  box-sizing: border-box;
+
+  width: $articlewidth;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  background-color: #fff;
+  color: #060b35;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  &__img {
+    padding-bottom: 40px;
+
+    &_flexed {
+      display: flex;
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    padding-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+
+    &_flex {
+      list-style: none;
+      display: flex;
+    }
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &:nth-child(-n + 4) {
+      background-image: url(/src/images/star-active.svg);
+    }
```

Comment:
You need to create a universal variant for 1,2,3,4,5 starsline. 
Try it. Thanks)